config/initializers/carrierwave_config.rb

```diff
@@ -10,7 +10,4 @@
     config.fog_directory = Settings.s3.upload_bucket
     config.fog_public = false
   end
-
-  UploadedFileUploader.storage = :fog
```

Comment:
Ah, because of this: https://github.com/projecthydra-labs/hybox/blob/d4462c4359fbb131a1e2520f1413c7d19888a653/config/initializers/carrierwave_config.rb#L5